### ISSUE TITLE
[$250] [Search v1] Create SearchPage

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#39880]
Section from [design doc](https://docs.google.com/document/d/1u6tBxmoKAEc70jvyZHaKykcfyC5iyEaewfgYXUSSbBc/edit#bookmark=id.4wp7kybymmno)
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~012c153c59b24c7284</li>
        <li>Upwork Job ID: 1780007609768833024</li>
        <li>Last Price Increase: 2024-04-15</li>
    </ul>
</details>

**Comment by Kicu** [comment]
@luacmartins since Im very close to being done with SearchFilters and I'd like to take this one next, can you assign me please.

**Comment by luacmartins** [comment]
Done!

**Comment by luacmartins** [comment]
@Kicu is still working on ti

**Comment by Kicu** [comment]
Just a small comment @luacmartins 
According to the [design doc](https://docs.google.com/document/d/1u6tBxmoKAEc70jvyZHaKykcfyC5iyEaewfgYXUSSbBc/edit#heading=h.o0ogvdg2aixu) most of the Search functionality will be captured within the `SearchWidget` and the List component. They are responsible for displaying the results, empty state, loading etc.
Furthermore the empty page skeleton PR is already open and navigation + routing is already done and merged.

Because of that I believe this task to be quite simple as it will consist mostly of composing components implemented in other PRs into a nice Page.
I plan to do this one at the very end - this is just connecting together existing blocks. I also expect this will not be a very big issue, even if the name might suggest otherwise.

**Comment by luacmartins** [comment]
Sounds good!

**Comment by luacmartins** [comment]
Still in dev

**Comment by luacmartins** [comment]
We'll get to this one once [the ListItem issue is done](https://github.com/Expensify/App/issues/39877) is done.

**Comment by luacmartins** [comment]
This was done as part of https://github.com/Expensify/App/pull/41102 and https://github.com/Expensify/App/pull/41347. Closing.

**Comment by luacmartins** [comment]
Payouts due:

- Contributor+: $250 @thesahindia for PR review

**Comment by JmillsExpensify** [comment]
$250 approved for @thesahindia
